### PR TITLE
Feat/Full text search

### PR DESCRIPTION
**Pull request by mugikhan**
Update MakeFile with Full Text Search (FTS5) targets.
Update shell script for building fts targets
Update README.md

### CODE REVIEW

File: src/types/index.d.ts [hunk 1]
```diff
@@ -15,7 +15,13 @@
  * each element converted to a byte); SQLite always returns blob data as
  * `Uint8Array`
  */
-type SQLiteCompatibleType = number | string | Uint8Array | Array<number> | bigint | null;
+type SQLiteCompatibleType =
```
Comment (stevensJourney) [inline src/types/index.d.ts:18]: Formatting here might make it difficult to keep in sync with the upstream repository. Speaking of which, it would be nice to pull in the latest changes from `upstream` before merging this. This can be in a separate PR.

File: Makefile [hunk 2]
```diff
@@ -51,7 +53,7 @@ EMFLAGS_COMMON = \
 	-s WASM=1 \
 	-s INVOKE_RUN \
 	-s ENVIRONMENT="web,worker" \
-	-s STACK_SIZE=512KB \
+	-s TOTAL_STACK=512KB \
```
[the remark this answers is not in the document]
Comment (mugikhan) [reply Makefile]: This was from when I was testing a different version of emscripten. I have reverted the change.

File: Makefile [hunk 3]
```diff
@@ -26,8 +26,10 @@ EXPORTED_RUNTIME_METHODS = src/extra_exported_runtime_methods.json
 ASYNCIFY_IMPORTS = src/asyncify_imports.json
 
 # intermediate files
-OBJ_FILES_DEBUG = $(patsubst %.c,tmp/obj/debug/%.o,$(CFILES))
-OBJ_FILES_DIST = $(patsubst %.c,tmp/obj/dist/%.o,$(CFILES))
+# OBJ_FILES_DEBUG = $(patsubst %.c,tmp/obj/debug/%.o,$(CFILES))
+# OBJ_FILES_DIST = $(patsubst %.c,tmp/obj/dist/%.o,$(CFILES))
+OBJ_FILES_DEBUG_FTS = $(patsubst %.c,tmp/obj/debug/%.o,$(CFILES))
```
Comment (stevensJourney) [inline Makefile:31]: I think these are still the same list of files with and without FTS enabled?

File: src/types/index.d.ts (hunk 1 above)
Comment (mugikhan) [reply src/types/index.d.ts]: I've reverted the changes

**Review by stevensJourney — APPROVED** [review]
Remember to add a `minor` changeset before merging
